include/sys/spa.h

```diff
@@ -555,13 +675,14 @@ _NOTE(CONSTCOND) } while (0)
 		    DVA_GET_ASIZE(&bp->blk_dva[1]) / 2)			\
 			copies--;					\
 		len += func(buf + len, size - len,			\
-		    "[L%llu %s] %s %s %s %s %s %s%c"			\
+		    "[L%llu %s] %s %s %s %s %s %s %s%c"			\
```

Comment:
hmm, I'm not sure how tight the 320 is currently.  Ideally we would figure out what the max length is and only increase it if needed.  But if that's a pain, then just bump it by the max that you're adding.